errors.go

```diff
@@ -22,13 +22,15 @@ import (
 	"strconv"
 	"strings"
 
+	"errors"
```

Comment:
Dependency on errors is not necessary if we wanted to keep backwards compatibility with 1.11 and 1.12